src/resources/apis.yml

```diff
@@ -27,45 +11,16 @@ apis:
     documentation: https://docs.botframework.com/en-us/
-    prizes: Use of this API is eligible for the Microsoft Cognitive Services and Bot Framework Prize!
-  - name: eLife API
-    description: |
-      [eLife](https://elifesciences.org/) is an online open-access journal for life sciences and biomedical research. Scientific manuscripts that have been
-      accepted after peer review are published online as articles available both in HTML and PDF format. Please note the eLife API documentation is still
-      under development; feedback welcome.
-    documentation:  https://github.com/elifesciences/api-raml
-    prizes: Use of this API is eligible for the eLife Challenge!
-  - name: eLife API SDK for PHP
-    description: |
-      SDK for building on eLife resources using PHP - see above for eLife API details. Eligible for the eLife Science API Prize.
-    documentation: https://github.com/elifesciences/api-sdk-php
-    prizes: Use of this API is eligible for the eLife Challenge!
-  - name: eLife Lens
-    description: |
-      Lens provides a novel way of looking at content on the web. It is designed to make life easier for researchers, reviewers, authors and readers. Lens is a
-      stand-alone web component that can be embedded into any web page.
-    documentation: https://github.com/elifesciences/lens
-    prizes: Use of this API is eligible for the eLife Challenge!
-  - name: eLife Continuum
-    description: |
-      eLife Continuum is the platform that we use to manage the publishing and hosting of our research content. It is composed of a set of software components
-      that form a publishing and article hosting system. Continuum has been made available for any interested party to use, to encourage the adoption of open online
-      continuous publishing. The platform can be used in its entirety, or users can exploit the individual components of Continuum within their own platform.
-    documentation: https://github.com/elifesciences/elife-continuum-documentation
-    prizes: Use of this API is eligible for the eLife Challenge!
-  - name: LastMile rasa NLU
-    description: |
-      [LastMile](http://golastmile.com/) provides AI dialogue technology entirely based on machine learning: developers don't hard-code conversation flow charts. Deep
-      reinforcement learning is used to automatically improve with every user interaction. The rasa NLU API provides open-source language understanding for bots.
-    documentation: https://rasa.ai/
   - name: Man AHL Arctic
     description: |
       [Man AHL](https://www.ahl.com/) are a leading quantitative investment manager in the UK and a pioneer in systematic trading for over 25 years. Their
       Arctic API provides a high performance datastore for time series and tick data.
     documentation: https://github.com/manahl/arctic
+  - name: TPP Example Public Health Data
+  - description: An example set of health record information in the CHAIR json format.
```

Comment:
Shouldn't this have a tab at the start, rather than a `-`?

Reply:
@varkor You're right, I've added [an issue](https://github.com/hackcambridge/hack-cambridge-website/issues/423) for YAML file validation